pympress/pointer.py

```diff
@@ -66,30 +85,49 @@ def __init__(self, config, builder):
         """
         super(Pointer, self).__init__()
         self.config = config
+        self.builder = builder
+
+        builder.load_widgets(self)
 
         self.redraw_current_slide = builder.get_callback_handler('redraw_current_slide')
 
-        default = 'pointer_' + config.get('presenter', 'pointer')
-        self.load_pointer(default)
+        default_mode = config.get('presenter', 'pointer_mode')
+        default_color = 'pointer_' + config.get('presenter', 'pointer')
+
+        # Adapt setting from old configuration
+        if default_color == 'pointer_none':
+            default_color = 'pointer_red'
+            default_mode  = 'none'
+            self.config.set('presenter', 'pointer', 'red')
+            self.config.set('presenter', 'pointer_mode', 'none')
```

Comment:
Can this be moved to the `Config` class? Maybe make an `upgrade` function to be called after reading the config in `Config.__init__`